dockstore-common/src/main/java/io/dockstore/common/metrics/ExecutionStatus.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.dockstore.common.metrics;
+
+public enum ExecutionStatus {
+    ALL, // Internal use only
+    SUCCESSFUL, FAILED, FAILED_SEMANTIC_INVALID, FAILED_RUNTIME_INVALID, ABORTED
```

Comment:
Could use javadoc for each state. Tangent, apparantly Markdown Javadoc is coming